src/hooks.server.ts

```diff
@@ -216,6 +216,10 @@ const handleGlobal: Handle = async ({ event, resolve }) => {
 		event.locals.email = session.email;
 		event.locals.npub = session.npub;
 		event.locals.sso = session.sso;
+		event.locals.clientIp = session.clientIp;
+		event.locals.countryCode = event.locals.clientIp
+			? countryFromIp(event.locals.clientIp)
+			: undefined;
```

Comment:
```suggestion
		if (session.pos?.countryCodeOverride) {
			event.locals.countryCode = session.pos.countryCodeOverride;
		};
```

I think that's what we want @ithiame . No need to store a fake IP, only the fake countrycode